### ISSUE TITLE
lualatex cannot find Fira Code

### ISSUE THREAD

**Issue by dotlambda** [NixOS/nixpkgs#118891]
**Describe the bug**
```
luaotfload | db : Reload initiated (formats: otf,ttf,ttc); reason: Font "FiraCode" not found.
luaotfload | resolve : sequence of 3 lookups yielded nothing appropriate.

! Package fontspec Error: The font "FiraCode" cannot be found.
```

```
$ fc-list | rg -i "fira code"
/home/rschuetz/.nix-profile/share/fonts/truetype/FiraCode-VF.ttf: Fira Code,Fira Code Light:style=Light,Regular
/home/rschuetz/.nix-profile/share/fonts/truetype/FiraCode-VF.ttf: Fira Code
/home/rschuetz/.nix-profile/share/fonts/truetype/FiraCode-VF.ttf: Fira Code:style=Bold
/home/rschuetz/.nix-profile/share/fonts/truetype/FiraCode-VF.ttf: Fira Code:style=SemiBold
/home/rschuetz/.nix-profile/share/fonts/truetype/FiraCode-VF.ttf: Fira Code Light:style=Retina
/home/rschuetz/.nix-profile/share/fonts/truetype/FiraCode-VF.ttf: Fira Code:style=Medium
/home/rschuetz/.nix-profile/share/fonts/truetype/FiraCode-VF.ttf: Fira Code,Fira Code Light:style=Regular
```

**To Reproduce**
Install the font with `nix-env -iA nixos.fira-code` and use it with `\setmonofont{Fira Code}`.

**Expected behavior**

**Notify maintainers**
@eyJhb @veprbl @rycee 

**Metadata**
```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.11.11, NixOS, 21.05pre-git (Okapi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.10`
 - channels(root): `"nixos-21.05pre281538.9e377a6ce42"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by veprbl** [comment]
This works for me
```tex
\usepackage{fontspec}

\setmainfont[Path=/nix/store/41qphrp1ksc1fqm01i7khs30n1zjl56r-fira-code-5.2/share/fonts/truetype/]{FiraCode-VF.ttf}
```

**Comment by dotlambda** [comment]
I discovered that `\setmonofont{Fira Code-VF}` also works. I think this is actually an issue with the `fira-code` package.
cc @rycee

**Comment by apfelkuchen6** [comment]
I can't reproduce the issue anymore. 
```tex
\documentclass{article}
\usepackage{fontspec}\setmonofont{Fira Code}
\begin{document}
Let's test the “programming ligatres” \verb|hello <> === ok nein|.
\end{document}
```
builds fine now. Looking at luaotfloads changelog, I suspect that the issue is fixed since the 3.19 release (which is contained in texlive-2022).
